### DISCUSSION TITLE
DNS check successful BUT my gitbub pages website is not working.

### DISCUSSION THREAD

**Discussion by zoulou421** [community/community#148251] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone! Please i need help. I bought a domain name from godaddy, i configured it successfully both on github and godaddy. But for now more than 24 hours, i can't see anything. My website is not showing. Nothing happened. I was assisted by godaddy, but they said that the problem is not on their side, it is a github issue. Please i need help

**Comment by NazimRexha123** [comment]
You should do over again from begining .It looks like Goddady has an issue with your domain .Ask them do you have to verify or ask to receive confirmation for your domain.
Best to you 
Naz

**Reply by zoulou421** [reply]
Great thanks. Yes, i have asked them. And multiple times they said to me that because my app is hosted on github pages, they are not able to solve the problem. Though my app hosted on github pages can work without a domain. But once i link my domain to my hosted github pages app, is not working at all.
